### DISCUSSION TITLE
Unobfuscate selected strings

### DISCUSSION THREAD

**Discussion by MrPlayerYork** [RevonZev/minecraft-python-mcpy#12] [Ideas]
It would be cool to have a little more control over what is obfuscated and what isn't.

for example, if you have a trigger scoreboard objective, maybe obfuscating that wouldn't be the best course of action cause then people would be /trigger 'ing a random scoreboard instead of what you want.

**Comment by revonzev** [comment]
I've added a way to disable auto obfuscation for scoreboard objectives [v2.4.0](https://github.com/RevonZev/minecraft-python-mcpy/releases/tag/v2.4.0) #13
